proto/binlogdata.proto

```diff
@@ -171,6 +171,11 @@ message Rule {
   // SourceUniqueKeyTargetColumns represents the names of columns in target table, mapped from the chosen unique
   // key on source tables (some columns may be renamed from source to target)
   string source_unique_key_target_columns = 7;
+
+  // ConvertIntToEnum lists any columns that are converted from an integral value into an enum.
+  // such columns need to have special transofrmation of the data, from an integral format into a
```

Comment:
Such a small nit that I'm annoying myself as I type 🤣 but we use sentence capitalization and punctuation in the first part and NOT in the second. IMO we should use it for both.

Reply:
Fixed